yt/fields/tests/test_fields_plugins.py

```diff
@@ -18,8 +18,8 @@ def myfunc():
 
 
 def setUpModule():
-    my_plugin_name = ytcfg.get("yt", "pluginfilename")
-    # In the following order if pluginfilename is: an absolute path, located in
+    my_plugin_name = ytcfg.get("yt", "pluginFilename")
```

Comment:
why was this case-change necessary ?

Reply:
There were some places in yt where `pluginfilename` was camel-cased and others where it was not, so I decided to use the camel-case version. This was no issue with the old config, but has now become one.

Reply:
Ok I get it, thanks !